test/test-mongocrypt-ctx-setopt.c

```diff
@@ -697,6 +697,119 @@ _test_setopt_for_explicit_encrypt (_mongocrypt_tester_t *tester)
    ALGORITHM_FAILS ("bad-algo", -1, "unsupported algorithm");
    EX_ENCRYPT_INIT_FAILS (bson, "unsupported algorithm");
 
+   /* It is an error to set the FLE 1 keyAltName option with any of the FLE 2
+    * options (index_type, index_key_id, or contention_factor). */
+   {
+      REFRESH
```

Comment:
Unrelated to PR, but `REFRESH` macro (and others in this file) should probably be adjusted at some point to consistently require a terminating semicolon:
```diff
- #define REFRESH                  \
-    mongocrypt_ctx_destroy (ctx); \
-    ctx = mongocrypt_ctx_new (crypt);
+ #define REFRESH                         \
+    do {                                 \
+       mongocrypt_ctx_destroy (ctx);     \
+       ctx = mongocrypt_ctx_new (crypt); \
+    } while (0)
```

Reply:
Good point. That is small enough to do here. Done.